PHPCI/Plugin/Email.php

```diff
@@ -70,12 +70,21 @@ public function execute()
 
         $buildStatus  = $this->build->isSuccessful() ? "Passing Build" : "Failing Build";
         $projectName  = $this->build->getProject()->getTitle();
-        $mailTemplate = $this->build->isSuccessful() ? 'Email/success' : 'Email/failed';
+        $mailTemplate = $this->build->isSuccessful() ? 'Email/long' : 'Email/short';
```

Comment:
Isn't it more logic to get a long email on failure and the short one on success? (Maybe that is just logic for me, I'm not sure)